infrastructure-playbooks/make-osd-partitions.yml

```diff
@@ -33,7 +33,7 @@
   tasks:
 
   - name: include regex vars
-    include_vars: ../roles/ceph-osd/defaults/regex.yml
+    include_vars: ../roles/ceph-defaults/defaults/main.yml
```

Comment:
Instead of using ``include_vars`` I think you could just add the ``ceph-defaults`` role to the playbook.

Above ``tasks`` add:

```
roles:
  - ceph-defaults
```